git-hg/updatetags8.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018 All Rights Reserved
+# ===========================================================================
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+#
+# ===========================================================================
+
+if [ $# -lt 2 ]; then
+   echo Usage: $0 '[jdk8u|jdk9] TAGS'
+   echo Version supplied should match a reposotiry in adoptopenjdk/openjdk-VERSION
+   echo And also http://hg.openjdk.java.net/VERSION/VERSION
```

Comment:
Not strictly true - the hg URLs are different....
jdk8u - http://hg.openjdk.java.net/jdk8u/jdk8u/
jdk9u - http://hg.openjdk.java.net/jdk-updates/jdk9u

Reply:
Thanks - will add in some more logic to handle that - haven't even looked at 9 yet

Reply:
Fixed